lib/pretty_text.rb

```diff
@@ -324,8 +324,11 @@ def self.extract_links(html)
     links = []
     doc = Nokogiri::HTML5.fragment(html)
 
-    # remove href inside quotes & elided part
-    doc.css("aside.quote a, .elided a").each { |a| a["href"] = "" }
+    # extract onebox links
+    doc.css("aside.onebox").each { |onebox| links << DetectedLink.new(onebox["data-onebox-src"], false) }
+
+    # remove href inside quotes & oneboxes & elided part
+    doc.css("aside.quote a, aside.onebox a, .elided a").each { |a| a["href"] = "" }
```

Comment:
Since we don't need those links, you could remove them from the DOM

```suggestion
    doc.css("aside.quote a, aside.onebox a, .elided a").remove
```